config/printer-lulzbot-mini2-2018.cfg

```diff
@@ -0,0 +1,138 @@
+#This file contains pin mappings for the stock Lulzbot Mini 2 which uses EinsyRetro mainboard and SingleExtruder(0.5mm) hotend.
+#To use this config, the firmware should be compiled for the AVR atmega2560.
+# See docs/Config_Reference.md for a description of parameters.
+# Pin numbers checked against Lulzbot fork of Marlin pins_EINSYRETRO.h
+# https://gitlab.com/lulzbot3d/marlin/-/blob/master/Marlin/src/pins/rambo/pins_EINSY_RETRO.h
+# validated against https://github.com/ultimachine/EinsyRetro/blob/1.0b/board/Project%20Outputs%20for%20EinsyRetro/Schematic%20Prints_EinsyRetro_1.0b.PDF
+[mcu]
+serial: /dev/ttyACM0
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 2000
+max_z_velocity: 40
+max_z_accel: 100
+[stepper_x]
+step_pin: PC0
+dir_pin: !PL0
+enable_pin: !PA7
+rotation_distance: 32
+microsteps: 16
+endstop_pin: tmc2130_stepper_x:virtual_endstop
+position_endstop: -7
+position_min: -7
+position_max: 168
+homing_positive_dir: FALSE
```

Comment:
This is not really needed, Klipper knows which direction to home according to the position_endstop value